app/controllers/users_controller.rb

```diff
@@ -11,16 +20,51 @@ def create
     redirect_to @user
   end
 
-  def show
-    @user = User.find_by id: params[:id]
-    return if @user
+  def show; end
 
-    flash[:danger] = t "error_user"
-    redirect_to root_url
+  def edit; end
+
+  def update
+    return render "edit" unless @user.update(user_params)
```

Comment:
```suggestion
    return render :edit unless @user.update(user_params)
```
Ưu tiên dùng sym nhé